cmd/configschema/go.mod

```diff
@@ -303,6 +303,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.47.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.47.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor v0.47.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/processor/schematransformerprocessor v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
I'm not sure about this change. Would it get updated to the same version as the others when they are bumped?

Reply:
Yes, that'll get updated with the next release prep PR after it is merged.